src/js/helpers/JSONPUtil.js

```diff
@@ -0,0 +1,44 @@
+import Q from "q";
+
+const JSONPUtil = {
+  /**
+   * Request JSONP data
+   *
+   * @todo: write test to verify this util is working properly
```

Comment:
Yep, but as long as we don't have an infrastructure featuring a real DOM, testing will be hard. We could extract some of it's functionality (e.g. building the url) into a util and unit test those methods. What do you think?